### PR TITLE
Fix Xcode 12 warnings

### PR DESCRIPTION
**Pull request by DenTelezhkin**
SPM in Xcode 12 bumps minimum iOS deployment target to iOS 9, which produces warnings when trying to use Dwifft.

This PR fixes that by deleting minimum supported platforms from Package.swift. This way minimum deployment targets are determined by build tools themselves.

Also fixed one Swift warning for Swift 5.3, and removed copyright header, which does not seem to be needed here :)

### CODE REVIEW

File: Source/Diff.swift [hunk 1]
```diff
@@ -26,7 +26,7 @@ public enum Operation: Equatable {
     case let (.insert(l), .insert(r)),
          let (.delete(l), .delete(r)),
          let (.update(l), .update(r)): return l == r
-    case let (.move(l), .move(r)): return l == r
+    case let (.move(l1,l2), .move(r1,r2)): return l1 == r1 && l2 == r2
```
Comment (mcudich) [inline Source/Diff.swift:29]: Is this change related to an Xcode warning?

Comment (DenTelezhkin) [reply Source/Diff.swift:29]: Yes, seems like a new Swift 5.3 warning about decomposing tuples.
